wiv.js

```diff
@@ -80,7 +94,7 @@ function wiv() {
   /**
   Represents the logic to draw a single frame. Animates all wivs
   */
-  function drawLines(canvas, speed, height, tightness, thickness, frame, color, ctx = null) {
+  function drawLines(canvas, speed, height, tightness, thickness, increment, frame, color, ctx = null) {
```

Comment:
@jjkaufman: I just added the increment value as another argument here, but I don't like how long this method signature is getting. What would you think of combining some of these into a generic params object? So it would looks something like `drawLines(canvas, ctx, params)`?

Reply:
yah I agree. Even before increment was added 😆 

We'll throw it in another PR